packages/migrate/src/__tests__/__snapshots__/DbPull.test.ts.snap

```diff
@@ -66,6 +66,160 @@ model jobs {
 }
 
 
+// introspectionSchemaVersion: NonPrisma,
+`;
+
+exports[`cockroachdb basic introspection (no schema) --url 2`] = `
+datasource db {
+  provider = "postgresql"
```

Comment:
Is this expected to be PostgreSQL?

Reply:
Yes, as we only use the provider protocol or the existing provider in the schema file.